common/views/components/SearchBox/README.md

```diff
@@ -0,0 +1,5 @@
+## Purpose
+
+To provide a means for users to find what they're looking for.
+
+A form containing a text input and a button.
```

Comment:
What does this sentence mean?

Reply:
Describing the implementation details. Do you think it would be better without it?

Reply:
Probably as those are likely to change.
And it doesn't help with making any further implementation details, e.g. when to use it etc.

Personally keeping docs abstract and idealogical means you probably don't need to update them often, and they don't risk going out of date.

Reply:
yep, makes sense